test/test_cabinet.py

```diff
@@ -151,6 +152,98 @@ def test_read_create(self):
         self.assertDictEqual(data, result)
         client.delete_bucket(Bucket=bucket)
 
+    def test_read_create_with_default_region(self):
+        client = boto3.client('s3', 'us-east-2')
```

Comment:
Is this actually the default region?

Reply:
Nope, and you cannot even create a Bucket using the default region so I am removing this test.